lib/internal/crypto/webcrypto.js

```diff
@@ -64,8 +64,11 @@ const {
 
 const {
   getRandomValues,
+  randomUUID: _randomUUID,
 } = require('internal/crypto/random');
 
+const randomUUID = () => _randomUUID();
```

Comment:
Why is this useful? Can't you pass directly the function from `internal/crypto/random`?

Reply:
For two reasons:
- `randomUUID.length` should be 0
- It should not be allowed to call this function with `new`